docs/reference/commandline/dockerd.md

```diff
@@ -191,28 +191,28 @@ communication with the daemon.
 > **Note**
 >
 > If you're using an HTTPS encrypted socket, keep in mind that only
-> TLS1.0 and greater are supported. Protocols SSLv3 and under are not
-> supported anymore for security reasons.
+> TLS version 1.0 and higher is supported. Protocols SSLv3 and below are not
+> supported for security reasons.
```

Comment:
Remind me; we need to verify what TLS versions we support now (it's very likely we no longer support 1.0)